ckanext/cfpb_extrafields/digutils.py

```diff
@@ -0,0 +1,94 @@
+from openpyxl import load_workbook
+try:
+    from ckan.plugins.toolkit import Invalid
+except ImportError: # pragma: no cover
+    #If the custom exception can't be imported, use a more generic exception
+    Invalid = Exception
```

Comment:
When we run the unit tests on travis, we don't install CKAN. The files that are called during unit tests (validators, digutils, exportutils) contain as much of the testable logic as possible without depending on CKAN to run.